pkg/webui/console/components/device-data-form/index.js

```diff
@@ -204,39 +237,56 @@ class DeviceDataForm extends Component {
           disabled={update}
           component={Input}
         />
+        <Form.Field
+          title={m.externalJoinServer}
+          description={m.externalJoinServerDescription}
+          name="external_js"
```

Comment:
Suggestion: I think it makes sense to mark fields that do not correspond to an entity value by prepending a `_`. Same thing for `activation_mode` then.
What do you think?

Reply:
I think this is not important